### PR TITLE
doc: add minutes for meeting 11 Sep 2107

### PR DESCRIPTION
**Pull request by mhdawson**
@nodejs/benchmarking please review/comment.

### CODE REVIEW

**Comment by mhdawson** [comment]
@nodejs/benchmarking please take a look

File: meetings/2017-09-11/minutes.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  * Reviewed PR for DC-EIS working
+  * Was working on #132 based on feedback 
+
+* Gareth Ellis (@gareth-ellis)
+  * Not very much - will turn doc on using benchmark job in #127 into a PR. 
+  * However people can still try out the job
+  
+* Uttam Pawar (@uttampawar)
+  * Integration DC-EIS of into the benchmarking environment
+  * Seems to be running ok, working on the templates the charts
```
Comment (kunalspathak) [inline meetings/2017-09-11/minutes.md]: nit: "templates of the chart"

File: meetings/2017-09-11/minutes.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  * Ramping up on the ansible infrastructure and will then work on the
+    updates for DC-EIS
+
+* Anil 
+
+* Benedict 
+  * working on issue 138, uploaded version 1 today, missing some parts that he wanted
```
Comment (kunalspathak) [inline meetings/2017-09-11/minutes.md]: nit: "issue # 138

**Comment by mhdawson** [comment]
Landed as 30c2d9ad46826de16fc2e9e0c160e28a0bae49af

**Pull request closed without merging**